packages/hop-node/src/watchers/SyncWatcher.ts

```diff
@@ -662,6 +711,212 @@ class SyncWatcher extends BaseWatcher {
 
     return true
   }
+
+  isOruToL1 (destinationChainId: number) {
+    const sourceChain = this.chainSlug
+    const destinationChain = this.chainIdToSlug(destinationChainId)
+    return destinationChain === Chain.Ethereum && bondableChains.includes(sourceChain)
+  }
+
+  isNonOruToL1 (destinationChainId: number) {
+    const sourceChain = this.chainSlug
+    const destinationChain = this.chainIdToSlug(destinationChainId)
+    return destinationChain === Chain.Ethereum && !bondableChains.includes(sourceChain)
+  }
+
+  // ORU -> L1: (credit - debit - OruToL1PendingAmount - OruToAllUnbondedTransferRoots) / 2
+  //    - divide by 2 because `amount` gets added to OruToL1PendingAmount
```

Comment:
nitpick: we divide by 2 in the comments but we don't implement the /2 here, we do it upstream. I think it makes sense to keep the whole equation here in the comments, but it might be worth adding another comment saying that the /2 happens upstream.

Reply:
good point, will do